### PR TITLE
Update messages_en.properties

### PR DESCRIPTION
**Pull request by lesgou**
1. Are line 13 and line 14 the same? If we are using the email as the user name, should we just use line 14?
2. Can we change line 24 to be more user friendly?
3. Is line 26 needed, since it is a generic version of the other invalid password error messages?

### CODE REVIEW

**Comment by hkeeler** [comment]
>Are line 13 and line 14 the same? If we are using the email as the user name, should we just use line 14?

I believe we can use just line 14, but we need to do a bit of testing.

>Can we change line 24 to be more user friendly?

@lesgou and I discussed this one.  We will not be using regex-based password rules, so this one can be ignored.

>Is line 26 needed, since it is a generic version of the other invalid password error messages?

I'll have to dig into that one.  It is likely a catch all if some error occurs on the backend during password validation.  It is probably unlike to show up, but we should keep it.

**Comment by hkeeler** [comment]
@lesgou and @keileyo, would you like me to merge this on as-is (once any questions have been answered), or do you want to add all of the other error message items we discussed earlier today onto this PR?

**Comment by hkeeler** [comment]
Also, I am currently unable to test this.  For whatever reason, I cannot get the institution test data to load in my dev env, which means I cannot complete the registration workflow.  @nickgrippin and I have been working on figuring out what's up.  As things stand now, these are very simple changes, so I'm pretty comfortable merging as is.  If we do include some of the other changes we discussed today, I'd like to wait to merge until I can test locally.

File: keycloak/themes/hmda/login/messages/messages_en.properties [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 unknownEmailDomainMessage=&#8226; The email domain <strong>{0}</strong> is not in the known list of HMDA filer domains.
-institutionErrorMessage=&#8226; An error occurred while validating the institution(s) against the <strong>{0}</strong> email domain.
-missingFirstNameMessage=&#8226; Please specify first name.
-missingLastNameMessage=&#8226; Please specify last name.
-missingEmailMessage=&#8226; Please specify email.
-missingUsernameMessage=&#8226; Please specify username.
-missingPasswordMessage=&#8226; Please specify password.
-missingTotpMessage=&#8226; Please specify authenticator code.
+institutionErrorMessage=&#8226; An error occurred while validating the financial institution(s) against the <strong>{0}</strong> email domain.
+missingFirstNameMessage=&#8226; Please specify the user's first name.
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: Do these need `''` like [here](https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/login/messages/messages_en.properties#L79)?

**Comment by keileyo** [comment]
@hkeeler once @awolfe76's questions have been answered, this one can be merged.  Agreed that we don't need to wait for answers to the questions discussed this AM.

File: keycloak/themes/hmda/login/messages/messages_en.properties [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 unknownEmailDomainMessage=&#8226; The email domain <strong>{0}</strong> is not in the known list of HMDA filer domains.
-institutionErrorMessage=&#8226; An error occurred while validating the institution(s) against the <strong>{0}</strong> email domain.
-missingFirstNameMessage=&#8226; Please specify first name.
-missingLastNameMessage=&#8226; Please specify last name.
-missingEmailMessage=&#8226; Please specify email.
-missingUsernameMessage=&#8226; Please specify username.
-missingPasswordMessage=&#8226; Please specify password.
-missingTotpMessage=&#8226; Please specify authenticator code.
+institutionErrorMessage=&#8226; An error occurred while validating the financial institution(s) against the <strong>{0}</strong> email domain.
+missingFirstNameMessage=&#8226; Please specify the user''s first name.
```
Comment (awolfe76) [inline keycloak/themes/hmda/login/messages/messages_en.properties]: I updated the `''` in https://github.com/cfpb/hmda-platform-auth/pull/83/commits/bae16484c9d11aca2af721521040c934a4bdf25a but do you think these would make more sense as `Please specify your first name.`?

**Pull request merged**